README.md

```diff
@@ -19,6 +19,19 @@ Things that are included:
 
 ## Getting started
 
+### Infrastructure
+Install `hedgehog`, `tasty`, `tasty-hedgehog`, and `tasty-hspec` to ease the development
+using [`ghcid`](https://github.com/ndmitchell/ghcid).
+
+```
+stack install ghcid hedgehog tasty tasty-hedgehog tasty-hspec
```

Comment:
You don't really need to `install` libraries. They're specified in `build-depends`, so they will be installed automatically after you execute `stack build` command.

Though, this is a good idea to `stack install ghcid` because there's dedicated `Makefile` for `ghcid` that allows you to run `make ide` command and run `ghcid` with proper options. I think `README.md` could mention this:

* https://github.com/Holmusk/three-layer/blob/38a629d350ca500c9ebd463d7a9af6205d679101/Makefile#L2